tests/nvidiagpu/deploy-gpu-test.go

```diff
@@ -54,10 +55,15 @@ var (
 	nfdCleanupAfterInstall bool = false
 
 	// NvidiaGPUConfig provides access to general configuration parameters.
-	nvidiaGPUConfig        *nvidiagpuconfig.NvidiaGPUConfig
-	gpuScaleCluster        bool = false
-	gpuCatalogSource            = "undefined"
-	gpuSubscriptionChannel      = "undefined"
+	nvidiaGPUConfig               *nvidiagpuconfig.NvidiaGPUConfig
+	gpuScaleCluster               bool = false
+	gpuCatalogSource                   = "undefined"
+	gpuSubscriptionChannel             = "undefined"
+	gpuDefaultSubscriptionChannel      = "undefined"
+	cleanupAfterTest              bool = false
```

Comment:
As discussed, I think this should be `true` by default, in order to avoid unexpected behavior when a cluster is reused.

Reply:
Changed default behavior to true by in next commit.